Composer/packages/client/src/components/Modal/AlertDialog.tsx

```diff
@@ -2,32 +2,19 @@
 // Licensed under the MIT License.
 
 /** @jsx jsx */
-import { jsx, css } from '@emotion/core';
+import { jsx } from '@emotion/core';
 import { Dialog, DialogType, DialogFooter } from 'office-ui-fabric-react/lib/Dialog';
 import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
 import ReactDOM from 'react-dom';
+import formatMessage from 'format-message';
 
-import { dialogStyle } from './dialogStyle';
+import { dialogStyle, NORMAL_STYLE, CONSOLE_STYLE } from './dialogStyle';
```

Comment:
I've been scrubbing through the code to merge styles into components rather than pick up styles from external modules.

Reply:
I just saw your email about it - I'll change these back, then.